### ISSUE TITLE
[Feature Request] Dismissible after X seconds

### ISSUE THREAD

**Issue by feelingsonice** [exyte/PopupView#198]
It would be nice to have a way to enable dismiss only after X seconds. Whether it be tap dismiss, drag dismiss, or something else :)

**Comment by f3dm76** [comment]
Hey @feelingsonice, it's a nice idea, I'll get to this as soon as I have time for features, have a wonderful day!

**Comment by VolleyZ** [comment]
I also need this method to dismiss, and i also hope to add method to dismiss without animation. Thanks author~~

**Comment by f3dm76** [comment]
Hey @feelingsonice, @VolleyZ, please try version 4.1.6, have a great day!
